src/routes/_layout.svelte

```diff
@@ -55,37 +56,51 @@
 <ScreenGauge devMode={dev} />
 <ColorCorrection options={$colorCorrectionOptions} />
 
-{#if $screen?.sizeFlags.medium}
+<section class={'usercontent ' + $screen?.classes} style={`--content-height: ${contentHeight}px`}>
```

Comment:
- I'd remove `usercontent` and I'd use `section` in the css
- I'd have both `class` and `style` as reactive vars
- it seems we're not using `--content-height` in the style